config/everything.mk

```diff
@@ -37,6 +37,7 @@ help:
 	# Explicit goals are: all bin include lib unit-test help clean distclean asm ppp
 	# "make all" is equivalent to "make bin include lib unit-test"
 	# "make bin" makes all binaries for the current platform
+	# "make ebpf-bin" makes all eBPF binaries
 	# "make include" makes all include files for the current platform
```

Comment:
Amusingly, this target is invalid until you get your first use of ebpf (in the next PR).    (Resolve this at will.. it is just a comment)